apps/src/applab/ChartApi.js

```diff
@@ -10,7 +10,7 @@
 var utils = require('../utils');
 var AppStorage = require('./appStorage');
 var GoogleChart = require('./GoogleChart');
-require("babelify/polyfill"); // required for Promises in IE / Phantom
+require("babel-polyfill"); // required for Promises in IE / Phantom
```

Comment:
we may want to move where we require this, but im going to leave that for a separate PR